content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -96,7 +101,90 @@ Once an override has been registered, it will exist until explicitly removed. To
 1. In Entity Outliner, **right-click** on an entity with overrides and choose **Revert Overrides** from the context menu.
 1. Notice that the entity no longer has any indication of having overrides.
 
-![Reverting overrides on an entity.](/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert.png)
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert.png" width="350" alt="Reverting overrides on an entity." >}}
+
+## Prefab overrides in Entity Inspector
+
+The above sections describe how you can manage prefab overrides in Entity Outliner, however the visualization support is limited to the entity level. This section further demonstrates prefab override functionality in Entity Inspector.
+
+Prefab override management in Entity Inspector is developed based on the new *Document Property Editor (DPE)*, which aims to replace the old *Reflected Property Editor (RPE)*.
+
+### Enable prefab overrides in Entity Inspector
+
+First, you need to enable the new DPE feature:
+1. In **O3DE Editor**, from the **Tools** menu, choose **Console Variables**.
+1. In Console Variables editor, set the `ed_enableDPE` flag to true.
+
+Alternatively, if you want the flag to be automatically enabled when O3DE Editor starts, you can create a settings registry file called `dpe.editor.setreg` with the following contents:
+
+```json
+{
+    "O3DE": {
+        "Autoexec": {
+            "ConsoleCommands": {
+                "ed_enableDPE": true
+            }
+        }
+    }
+}
+```
+
+{{< note >}}
+If you manually enable or disable the DPE through the Console Variables editor, you might need to close and reopen the Entity Inspector.
+{{< /note >}}
+
+Second, to enable prefab overrides in inspector, you can create a settings registry file called `editorpreferences.setreg` with the following contents:
```

Comment:
This can actually be set in any settings registry file. Not just the 'editorpreferences.setreg'.